crosstool/BUILD.tpl

```diff
@@ -41,6 +41,13 @@ cc_toolchain_suite(
     toolchains = dict(CC_TOOLCHAINS),
 )
 
+filegroup(
+    name = "modulemap",
+    srcs = [
+%{real_modulemap}
+    ],
+)
```

Comment:
Why do you go through a `filegroup` instead of referencing it directly?

Reply:
otherwise we'd need 2 different substitutions because for the osx_tools target we would have to substitute to nothing, and for the cc_toolchain_config target we'd have to either substitute to `None` or `module_map = thing`